### PR TITLE
Troubleshoot docker hangs

### PR DESCRIPTION
**Pull request by DonSchenck**
`docker foo` can hang. This explains how to fix it.

### CODE REVIEW

File: _references/troubleshooting-docker-hangs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Troubleshooting: My Docker command hangs
+seo: docker hangs
+---
```
Comment (kmsholcomb) [inline _references/troubleshooting-docker-hangs.md]: Revise the metadata fields as follows:

title: Docker command stops responding
author: Don Schenck <don.schenck@rackspace.com>
date: 2015-10-22
permalink: docs/references/troubleshooting-docker-hangs/
description: Resolve the issue of the docker command not responding
docker-versions:
  - 1.8.2
topics:
  - docker
  - troubleshooting

File: _references/troubleshooting-docker-hangs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+`docker ps -a`
+
+To fix this issue, perform the following steps:
+1. Press Ctrl-C to kill the `docker` command.
+2. Log in to your RCS account to view the cluster you are currently using.
+3. Click the icon to rebuild your cluster.
+
+![Example clusters]({% asset_path troubleshoot-docker-hangs/clusters.png %})
+4. When the rebuild is complete (you may need to click the "Refresh" button to update your screen), you can resume using the cluster.
```
Comment (kmsholcomb) [inline _references/troubleshooting-docker-hangs.md]: All of my edits for the content are as follows:

When you are using Carina, the `docker` command might stop responding. That is, you run a command, such as the following one, and nothing happens:

`docker ps -a`

To fix this issue, perform the following steps: 

1. Press **Ctrl-C** to kill the command. 

2. Log in to your Carina account to view the cluster that you are currently using. 

3. Rebuild your cluster.

    [Example cluster screenshot][If you don't have this finalized, just omit it.]

4. When the rebuild is complete (you might need to click the **Refresh** button to update your screen), you can resume using the cluster.

**Comment by kmsholcomb** [comment]
Hey @DonSchenck: I left two comments in the file, one for the metadata fields and one for the content. Please let me know if you have any questions. Thanks!

**Comment by everett-toews** [comment]
@DonSchenck Is this still valid?

Should it perhaps be closed and maybe resubmitted later?

**Comment by kimtryce** [comment]
Closing this PR.

**Pull request closed without merging**